react-version/src/components/context/BoardContext.js

```diff
@@ -0,0 +1,49 @@
+import React, { createContext, useContext, useState } from "react";
+import { CharaterContext } from "./CharContext";
+import char1 from "../../img/char2.png";
+import char2 from "../../img/char3.png";
+import char3 from "../../img/char4.png";
+import char4 from "../../img/char5.png";
+import char5 from "../../img/char6.png";
+
+const boardData = {
+	btnId: null,
```

Comment:
Which button? What is it's purpose? Would it be there only one button on board? I suggest to use more verbose naming for such things to let other understand it better.